### PR TITLE
Update Nvidia sai.profile SKU files to use common file

### PR DESCRIPTION
**Pull request by noaOrMlnx**
Add new common sai.profile,
Update Nvidia/Mellanox sai.profile of all SKUs to remove common parameters.
#### Why I did it
To have the ability to add common parameters to only 1 file instead of all SKUs.

### CODE REVIEW

**Review by nazariig — CHANGES_REQUESTED** [review]

File: platform/mellanox/docker-syncd-mlnx/sai-profile-common [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
+SAI_DUMP_STORE_AMOUNT=10
+SAI_DUMP_MFT_CFG_PATH=/etc/mft/fwtrace_cfg
```
Comment (nazariig) [inline platform/mellanox/docker-syncd-mlnx/sai-profile-common:3]: @noaOrMlnx do we need the same for `syncd` RPC docker? What about VS docker?

**Review by nazariig — CHANGES_REQUESTED** [review]

File: platform/mellanox/docker-syncd-mlnx/Dockerfile.j2 [hunk 2]
```diff
@@ -65,6 +65,7 @@ COPY ["supervisord.conf.j2", "/usr/share/sonic/templates/"]
 COPY ["files/supervisor-proc-exit-listener", "/usr/bin"]
 COPY ["critical_processes", "/etc/supervisor/"]
 COPY ["platform_syncd_dump.sh", "/usr/bin/"]
+COPY ["sai-profile-common", "/usr/bin/"]
```
[the remark this answers is not in the document]
Comment (noaOrMlnx) [reply platform/mellanox/docker-syncd-mlnx/Dockerfile.j2:68]: Done

File: device/mellanox/x86_64-mlnx_msn2740-r0/ACS-MSN2740/sai.profile [hunk 3]
```diff
@@ -1,3 +1 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_2740.xml
```
Comment (nazariig) [inline device/mellanox/x86_64-mlnx_msn2740-r0/ACS-MSN2740/sai.profile:1]: @noaOrMlnx is it by intention to inject `SAI_DUMP_MFT_CFG_PATH` to SKUs like this?

File: platform/mellanox/docker-syncd-mlnx/Dockerfile.j2 (hunk 2 above)
[the remark this answers is not in the document]
Comment (nazariig) [reply platform/mellanox/docker-syncd-mlnx/Dockerfile.j2]: @noaOrMlnx please mark all fixed comments as resolved

File: device/mellanox/x86_64-mlnx_msn2740-r0/ACS-MSN2740/sai.profile (hunk 3 above)
Comment (noaOrMlnx) [reply device/mellanox/x86_64-mlnx_msn2740-r0/ACS-MSN2740/sai.profile:1]: Yes

**Pull request closed without merging**